### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
🏋️‍♀️

### CODE REVIEW

File: runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs [hunk 1]
```diff
@@ -139,16 +139,14 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
 	/// The range of component `t` is `[500, 1000]`.
 	/// The range of component `a` is `[500, 800]`.
 	/// The range of component `d` is `[200, 400]`.
-	fn submit_unsigned(v: u32, t: u32, a: u32, d: u32, ) -> Weight {
-		(0 as Weight)
+	fn submit_unsigned(v: u32, _t: u32, a: u32, d: u32, ) -> Weight {
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs:142]: Big jump from 0 to 196_420_000

File: runtime/polkadot/src/weights/pallet_election_provider_multi_phase.rs [hunk 2]
```diff
@@ -109,11 +109,11 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
 	/// The range of component `a` is `[500, 800]`.
 	/// The range of component `d` is `[200, 400]`.
 	fn elect_queued(a: u32, d: u32, ) -> Weight {
-		(14_945_000 as Weight)
-			// Standard Error: 8_000
-			.saturating_add((909_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 13_000
-			.saturating_add((160_000 as Weight).saturating_mul(d as Weight))
+		(113_324_000 as Weight)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_election_provider_multi_phase.rs:112]: 10x jump of the weight from 14_945_000 to 113_324_000

File: runtime/rococo/src/weights/runtime_common_crowdloan.rs [hunk 3]
```diff
@@ -61,24 +61,24 @@ impl<T: frame_system::Config> runtime_common::crowdloan::WeightInfo for WeightIn
...
-		(50_967_000 as Weight)
+		(49_546_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `k` is `[0, 500]`.
 	fn refund(k: u32, ) -> Weight {
-		(0 as Weight)
+		(2_505_000 as Weight)
```
Comment (EgorPopelyaev) [inline runtime/rococo/src/weights/runtime_common_crowdloan.rs:79]: Big jump from 0

File: runtime/westend/src/weights/pallet_scheduler.rs [hunk 4]
```diff
@@ -77,9 +77,9 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	// Storage: Preimage StatusFor (r:1 w:1)
 	/// The range of component `s` is `[1, 50]`.
 	fn on_initialize_periodic_resolved(s: u32, ) -> Weight {
-		(2_421_000 as Weight)
+		(0 as Weight)
```
Comment (EgorPopelyaev) [inline runtime/westend/src/weights/pallet_scheduler.rs:80]: big jump down from 2_421_000 to 0

**Review by EgorPopelyaev — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]
Looks good on [SWC](https://weights.tasty.limo/compare-mr).  
Only one actually up 13.6% is `kusama cancel_deferred_slash`, but that should be fine.

**Pull request merged**